### PR TITLE
Update for BTJSON Refactor

### PR DESCRIPTION
**Pull request by sestevens**
**See [PR #675](https://github.com/braintree/braintree_ios/pull/675) in Braintree iOS**

### Summary of changes

 - There's one spot in Drop-in that needs to be updated once the BTJSON refactor in Braintree iOS is released.

**NOTE:** We'll need to pin Drop-in (CocoaPods and SPM) to use at least the new version of Braintree iOS once it's released.

 ### Checklist

 - ~Added a changelog entry~

- @sestevens 
- @scannillo

### CODE REVIEW

**Comment by bennibau** [comment]
I am using the latest Dropin and the App crashes because of this missing refactor.
https://github.com/braintree/braintree-ios-drop-in/issues/331
Please fix this asap. I cannot update my app otherwise

**Review by sshropshire — APPROVED** [review]
🌊

File: Package.resolved [hunk 1]
```diff
@@ -5,9 +5,9 @@
         "package": "Braintree",
         "repositoryURL": "https://github.com/braintree/braintree_ios",
         "state": {
-          "branch": null,
-          "revision": "6a1a28421b1ce65ec9a3cd834f1e5794c3a85d7d",
-          "version": "5.3.0"
+          "branch": "btjson-refactor",
```
Comment (sarahkoop) [inline Package.resolved:8]: Should this be branch null and version 5.3.2?

**Review by sarahkoop — APPROVED** [review]

File: Package.resolved (hunk 1 above)
Comment (scannillo) [reply Package.resolved:8]: Yes totally. Good catch!

**Review by demerino — APPROVED** [review]

**Pull request merged**